src/pip/_internal/commands/search.py

```diff
@@ -140,9 +140,7 @@ def print_results(hits, name_column_width=None, terminal_width=None):
                 summary = ('\n' + ' ' * (name_column_width + 3)).join(
                     summary_lines)
 
-        line = '{name_latest:{name_column_width}} - {summary}'.format(
-            name_latest='{name} ({latest})'.format(**locals()),
-            **locals())
+        line = f'{{{name} ({latest}):{name_column_width}}} - {summary}'
```

Comment:
I don’t think this works?

Reply:
Good catch. I misunderstood what the code is trying to do. I'll fix it